resources/config.edn

```diff
@@ -0,0 +1,76 @@
+{:datasets [{:indicator-id "23"
+             :indicator-field "Indicator value"
+             :conditions [{:field "Level description" :value "England"}]
```

Comment:
caps and spaces in strings that you are trying to match in data worry me. downcase and join with _ usually unless there is a reason not to.